### PR TITLE
fix: popup closes on context menu copy

### PR DESCRIPTION
**Pull request by astandrik**
Closes https://github.com/ydb-platform/ydb-embedded-ui/issues/1452

[Stand](http://ydb-vla-dev02-005.search.yandex.net:8765/monitoring/cluster/storage)

Problem: 

https://github.com/user-attachments/assets/64e2752f-05e4-4364-bd71-8fe10d0f6628

### Test Status: <span style="color: orange;">⚠️ FLAKY</span>
📊 [Full Report](https://ydb-platform.github.io/ydb-embedded-ui/1453/)

| Total | Passed | Failed | Flaky | Skipped |
|:-----:|:------:|:------:|:-----:|:-------:|
| 134 | 132 | 0 | 2 | 0 |

### Bundle Size: 🔺
Current: 78.96 MB | Main: 78.95 MB
Diff: +0.00 MB (0.01%)

⚠️ Bundle size increased. Please review.

<details>
<summary>ℹ️ CI Information</summary>

- Test recordings for failed tests are available in the full report.
- Bundle size is measured for the entire 'dist' directory.
- 📊 indicates links to detailed reports.
- 🔺 indicates increase, 🔽 decrease, and ✅ no change in bundle size.
</details>

### CODE REVIEW

**Comment by astandrik** [comment]
@Raubzeug @artemmufazalov ping

File: src/components/HoverPopup/HoverPopup.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+@import '../../styles/mixins.scss';
```
Comment (Raubzeug) [inline src/components/HoverPopup/HoverPopup.scss:1]: seems to be redundant

File: src/components/HoverPopup/HoverPopup.tsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    const open = isPopupVisible || showPopup || isPopupContentHovered || isFocused;
+
+    return (
+        <React.Fragment>
+            <div ref={anchor} onMouseEnter={onMouseEnter} onMouseLeave={onMouseLeave}>
+                {children}
+            </div>
+            <Popup
+                contentClassName={b()}
+                anchorRef={anchorRef || anchor}
```
Comment (Raubzeug) [inline src/components/HoverPopup/HoverPopup.tsx:96]: Should we allow to pass `anchorRef`? Won't it break behaviour set by us?

File: src/components/HoverPopup/HoverPopup.tsx [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+    const onPopupBlur = React.useCallback(() => {
+        setIsFocused(false);
+    }, []);
+
+    const onPopupEscapeKeyDown = React.useCallback(() => {
+        setIsFocused(false);
+        setIsPopupContentHovered(false);
+        hidePopup();
+    }, [hidePopup]);
```
Comment (Raubzeug) [inline src/components/HoverPopup/HoverPopup.tsx:85]: do we really need all these `useCallback`?

**Review by Raubzeug — CHANGES_REQUESTED** [review]

File: src/components/HoverPopup/HoverPopup.tsx (hunk 2 above)
Comment (astandrik) [reply src/components/HoverPopup/HoverPopup.tsx:96]: We allow to pass anchorRef for custom anchors if they are needed
if anchorRef is not passed - default anchor from component is used

File: src/components/HoverPopup/HoverPopup.tsx (hunk 3 above)
Comment (astandrik) [reply src/components/HoverPopup/HoverPopup.tsx:85]: yes
because we pass this function as prop to external component which may use this function as prop

**Review by Raubzeug — APPROVED** [review]

**Pull request merged**